### PR TITLE
Emphasize the Commands on the "Working with Ballerina Modules" Page

### LINKED ISSUE #2030: Hard to find module creation code
**Description:**
It is hard to find important commands like `bal add <...>` in  [1](https://ballerina.io/learn/user-guide/structuring-ballerina-code/working-with-ballerina-modules/#creating-ballerina-modules) at a glance. Could improve the experience if these were emphasized.

**Suggested Labels:**
Version/SLAlpha3

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Emphasize the commands on the "Working with Ballerina Modules" page.
> Fixes #2030

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md [hunk 1]
```diff
@@ -95,7 +99,13 @@ Follow the steps below to write a simple program in a Ballerina package, which p
...
 3. Navigate to the created `helloworld` package directory.
 
-4. Execute the `bal build` command to build an executable of this file. You view the output below.
+4. Execute the command below to build an executable of this file. You view the output below.
+
+    ```bash
+    bal build
+    ```
+
+    You view the output below.
```
Comment (NipunaRanasinghe) [inline learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md:108]: "You view the output below" segment is repeated here.

File: learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md [hunk 2]
```diff
@@ -105,7 +115,13 @@ Follow the steps below to write a simple program in a Ballerina package, which p
...
     ```
 
-5. Execute the `bal run` command to run the program. You view the output below.
+5. Execute the command below to run the program. You view the output below.
+
+    ```bash
+    bal run
+    ```
+
+    You view the output below.
```
Comment (NipunaRanasinghe) [inline learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md:124]: "You view the output below" segment is repeated here.

File: learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md (hunk 1 above)
Comment (praneesha) [reply learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md]: Fixed.

File: learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md (hunk 2 above)
Comment (praneesha) [reply learn/user-guide/structuring-ballerina-code/structuring-ballerina-code.md]: Fixed.

**Comment by suleka96** [comment]
LGTM

**Review by suleka96 — APPROVED** [review]

File: learn/user-guide/structuring-ballerina-code/working-with-ballerina-modules.md [hunk 3]
```diff
@@ -27,9 +27,15 @@ A module name can contain one or more Ballerina identifiers separated by dots (.
 
 The value of the first identifier of the modules belonging to the same package will always be the package name.
 
-For example, let’s create a package called `winery` to demonstrate Ballerina modules. This package exposes an API to list the available wine types and to get details about a particular wine. You can organize the source code with three modules named `winery`, `winery.model`, and `winery.storage`.
+## Creating the Package
 
->**Tip:** If you are still inside the previous `helloword` directory, use the `cd ..` command to go to the parent directory. Then, execute the `bal new winery` command to create the `winery` package. 
+Execute the command below to create a package called `winery` to demonstrate Ballerina modules. 
+
+>**Info:** This package exposes an API to list the available wine types and to get details about a particular wine. Inside that, you can organize the source code with three modules named `winery`, `winery.model`, and `winery.storage`.
```
Comment (azinneera) [inline learn/user-guide/structuring-ballerina-code/working-with-ballerina-modules.md:34]:
```suggestion
>**Info:** This is an application package that lists the available wine types and to get details about a particular wine. Inside that, you can organize the source code with three modules named `winery`, `winery.model`, and `winery.storage`.
```

**Review by azinneera — APPROVED** [review]
Ballerina Packages changes LGTM

**Pull request merged**